Align element documentation with implementations (#15324)
PrairieLearn/PrairieLearn: docs/elements/pl-dropdown.md

## Customizations

| Attribute      | Type    | Default | Description                                                                                                                                                                                                                                          |
| -------------- | ------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | `allow-blank`  | boolean | false   | Whether an empty submission is allowed. By default, empty dropdowns will not be graded (invalid format). Setting `allow-blank="true"` requires `blank="true"`, since blank submissions cannot be allowed while the blank dropdown entry is disabled. |

Reviewer: I don't think the rationale is necessary, it is rarely listed in the rest of the documentation. Also, `blank="true"` is the default, and the phrasing makes it sound like it must be explicitly set. This phrasing more closely matches other descriptions of restrictions.

```suggestion
| `allow-blank`  | boolean | false   | Whether an empty submission is allowed. By default, empty dropdowns will not be graded (invalid format). Cannot be set to `true` when `blank="false"`. |
```

Author: Resolved in dc98012de7 — applied your wording.

*- Claude*

| Attribute      | Type    | Default | Description                                                                                                                                                          |
| -------------- | ------- | ------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `allow-blank`  | boolean | false   | Whether an empty submission is allowed. By default, empty dropdowns will not be graded (invalid format). Cannot be set to `true` when `blank="false"`.               |
| `answers-name` | string  | —       | Variable name to store data in. Note that this attribute has to be unique within a question, i.e., no value for this attribute should be repeated within a question. |

> | `aria-label`   | string  | None    | An accessible label for the element.                                                                                                                                 |

Reviewer 2: ```suggestion
| `aria-label`   | string  | —       | An accessible label for the element.                                                                                                                                 |
```

Author: Resolved in db5f0787 — switched the default `None` → `—`, and (matching the other input elements) the wording to "An accessible label for the input." Also fixed the other stray `None` defaults (pl-drawing `aria-label`/`aria-description`).

*- Claude*
